func main() {
  var a;
  foo("entered function");
}

func foo(a) {
  print(a);
  var a;
}

/*
*OUT*
entered function
ErrorType.NAME_ERROR
*OUT*
*/